archive/2025-09-14-vaultgemma-the-world-s-most-capable-differentially-private-llm.md

```diff
@@ -0,0 +1,200 @@
+
+## Applying the scaling laws to build VaultGemma
+
+The [Gemma](https://deepmind.google/models/gemma/) models are designed with responsibility and safety at their core. This makes them a natural foundation for developing a production-quality, DP-trained model like VaultGemma.
+
+### Algorithmic advancements: Training at scale
+
+The scaling laws we derived above represent an important first step towards training a useful Gemma model with DP. We used the scaling laws to determine both how much compute we needed to train a compute-optimal 1B parameter Gemma 2-based model with DP, and how to allocate that compute among batch size, iterations, and sequence length to achieve the best utility.
+
+One prominent gap between the research underlying the scaling laws and the actual training of VaultGemma was our handling of [_Poisson sampling_](https://en.wikipedia.org/wiki/Poisson_sampling), which is a central component of [DP-SGD](https://arxiv.org/abs/1607.00133). We initially used a straightforward method of loading data in uniform batches but then switched to Poisson sampling to get the best privacy guarantees with the least amount of noise. This method posed two main challenges: it created batches of different sizes, and it required a specific, randomized order for processing the data. We solved this by using our recent work on [Scalable DP-SGD](https://arxiv.org/abs/2411.04205), which allows us to process data in fixed-size batches — either by adding extra padding or trimming them — while still maintaining strong privacy protections.
+
+## Results
+
+Armed with our new scaling laws and advanced training algorithms, we built VaultGemma, to date the largest (1B-parameters) open model fully pre-trained with differential privacy with an approach that can yield high-utility models.
+
+From training VaultGemma, we found our scaling laws to be highly accurate. The final training loss of VaultGemma was remarkably close to what our equations predicted, validating our research and providing the community with a reliable roadmap for future private model development.
+
+![VaultGemma4_Performance](https://storage.googleapis.com/gweb-research2023-media/images/VaultGemma4_Performance.width-1250.png)
```

Comment:
"The results quantify the current resource investment required for privacy and demonstrate that modern DP training yields utility comparable to non-private models from roughly five years ago."